### PR TITLE
Implemented error handling for views

### PR DESCRIPTION
**Pull request by swethasrihari**

### CODE REVIEW

**Review by deepika-pdx — CHANGES_REQUESTED** [review]
User.js : try block should include schema.validate(data) as the error can occur while validating the data against the specified schema.

File: views/src/components/DateTime.js
[the remark this answers is not in the document]
Comment (swethasrihari) [reply views/src/components/DateTime.js:1]: Removed.

File: views/src/components/Exercise.js
[the remark this answers is not in the document]
Comment (swethasrihari) [reply views/src/components/Exercise.js:1]: Removed.

[the remark this answers is not in the document]
Comment (swethasrihari) [reply views/src/components/Exercise.js:1]: Included schema.validate(data) inside the try block

**Review by deepika-pdx — APPROVED** [review]
Now changes look good.

**Pull request merged**